.github/workflows/update-tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Update extension test
+on:
+  push:
+    branches-ignore:
+      - master
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: styfle/cancel-workflow-action@0.11.0
+      - run: sudo chown -R $USER:$USER $GITHUB_WORKSPACE
```

Comment:
Seems like this `chown` is not actually needed. It was in `tests.yml`, but since we run `docker compose` directly it's not needed anymore, so I removed it (https://github.com/Keeper-Wallet/Keeper-Wallet-Extension/pull/667/commits/cd4e8e13976f213620f1e743d0ac19f1fc422483)